acbuild/write.go

```diff
@@ -33,7 +34,8 @@ var (
 func init() {
 	cmdAcbuild.AddCommand(cmdWrite)
 
-	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite output ACI")
+	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite the resulting ACI")
+	cmdWrite.Flags().BoolVar(&compress, "compress", false, "compress the resulting ACI")
```

Comment:
Ship it

On Wed, Oct 21, 2015 at 1:42 PM, Derek Gonyeo <notifications@github.com>
wrote:

> In acbuild/write.go
> <https://github.com/appc/acbuild/pull/47#discussion_r42679494>:
>
> > @@ -33,7 +34,8 @@ var (
> >  func init() {
> >  	cmdAcbuild.AddCommand(cmdWrite)
> >
> > -	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite output ACI")
> > +	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite the resulting ACI")
> > +	cmdWrite.Flags().BoolVar(&compress, "compress", false, "compress the resulting ACI")
>
> I'm in favor of dropping support for skipping compression.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/acbuild/pull/47/files#r42679494>.
>